cucumber-jparest/pom.xml

```diff
@@ -50,23 +54,22 @@
 
         <dependency>
             <groupId>com.vladmihalcea</groupId>
-            <artifactId>hibernate-types-55</artifactId>
-            <version>2.19.2</version>
+            <artifactId>hibernate-types-60</artifactId>
+            <version>2.21.1</version>
             <scope>test</scope>
         </dependency>
 
         <!-- SpringDoc -->
         <dependency>
             <groupId>org.springdoc</groupId>
-            <artifactId>springdoc-openapi-ui</artifactId>
-            <version>1.6.5</version>
+            <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+            <version>2.0.2</version>
             <scope>test</scope>
         </dependency>
 
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
-            <scope>test</scope>
```

Comment:
Changing the scope of this is unexpected (unless I see why later in the review) in the cucumber-jparest it was initially only needed for tests

Reply:
Just for context, after the upgrade to Spring Boot 3, the scope of nested `jackson-datatype-jsr310` dependency started resolving to `test` instead of `compile` (see dependency tree excerpts below).
  
Before the upgrade to Spring Boot 3:
```
uk.gov.homeoffice.digital.sas:cucumber-jparest:jar:0.0.19
 +- org.springframework.boot:spring-boot-starter-web:jar:2.6.2:test
[INFO] |  +- org.springframework.boot:spring-boot-starter-json:jar:2.6.2:test
[INFO] |  |  +- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:jar:2.13.1:COMPILE
```

After the upgrade to Spring Boot 3
```
uk.gov.homeoffice.digital.sas:cucumber-jparest:jar:0.1.0
[INFO] +- org.springframework.boot:spring-boot-starter-web:jar:3.0.2:test
[INFO] |  +- org.springframework.boot:spring-boot-starter-json:jar:3.0.2:test
[INFO] |  |  +- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:jar:2.14.1:TEST
```
As a result, JavaTimeModule class couldn't be imported in production anymore; this is how I ended up removing `test` scope. Now I admit this was a lazy solution. Instead, I should explicitly import the `jackson-datatype-jsr310` dependency with compile scope. Corresponding change has been pushed